docs/user/sbt.rst

```diff
@@ -96,23 +96,23 @@ Scala Native supports two distinct linking modes:
 Garbage collectors
 ------------------
 
-1. **boehm.** (default)
+1. **immix.** (default, since 0.3)
```

Comment:
`1. **immix.** (default since 0.3.7, introduced in 0.3)`

Reply:
@densh I think this needs to say `0.3.8` (the next release) so the next below should be `through 0.3.7`

Reply:
My bad, you're absolutely correct, it should have been `default since 0.3.8` here and `default up until 0.3.7` for boehm.

Reply:
I used `through 0.3.7` as I think it is a bit more precise. We used boehm as the default version through `0.3.7` and then switched the default to immix for `0.3.8`. Hopefully you agree.

Reply:
Sounds good to me.

Reply:
I appreciate your attention to detail in these matters. Not only is documentation important but this also is a record of what happened.